packages/bot-skeleton/src/scratch/dbot.js

```diff
@@ -66,6 +69,8 @@ class DBot {
                 this.addBeforeRunFunction(this.checkForErroredBlocks.bind(this));
                 this.addBeforeRunFunction(this.checkForRequiredBlocks.bind(this));
 
+                initErrorHandlingListener('keydown', globalObserver);
```

Comment:
Please confirm if this event is only initializing for the bot builder tab or for every other tab? Also is there any way we can only initialize this when a block is selected as only then the deleting can happen?

Reply:
done ✅